src/client.rs

```diff
@@ -542,7 +547,7 @@ pub(crate) unsafe extern "C" fn native_oauth_refresh_cb<C: ClientContext>(
         }
     };
 
-    let errstr = match CString::new(vec![0_u8; token_info.errstr_size]) {
+    let errstr = match CString::new(vec![u8::MAX; token_info.errstr_size]) {
```

Comment:
[from documentation](https://doc.rust-lang.org/std/ffi/struct.CString.html#method.new) `CString::new` returns an error if the bytes contain a `0`, causing this part of the code to only error.

Reply:
`from_vec_unchecked` may be a better choice. I would think that we _want_ the vector to have only zeros.